package.json

```diff
@@ -35,5 +47,20 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@storybook/addon-actions": "^6.5.16",
```

Comment:
it is better to keep versions of installed libs as constant, please remove "^"

Reply:
Done